numpy/_core/tests/test_regression.py

```diff
@@ -457,24 +457,20 @@ def test_pickle_py2_bytes_encoding(self):
 
         test_data = [
             # (original, py2_pickle)
-            (np.str_('\u6f2c'),
```

Comment:
All these regression tests should pass without the paths being changes, can you revert these changes?

Reply:
Looking at these binary contents, they didn't change in my opinion. 
One of your suggestions was to reformat these binary strings into a single line with `# noqa` at the end - these ones are just reformatted. Should I make them multiline?

Reply:
Ugh, oops, I'm sorry I misread the diff, disregard this comment!